style/js/shield_defs.js

```diff
@@ -785,6 +823,29 @@ export function loadShields(shieldImages) {
       },
     };
   });
+  [
+    "FJ:Ningde",
+    "GD:Gaozhou",
+    "GD:Dianbai",
+    "GD:Huazhou",
+    "GD:Maoming",
+    "GD:Maonan",
+    "GD:Xinyi",
+    "HA:Luoyang",
+    "HA:Mengjin",
+    "JS:Liyang",
+    "JS:Xuzhou",
+    "JS:Wuzhong",
```

Comment:
It would be quite inconvenient to track all the counties, districts, and villages in China. Local route signs in China have a uniform appearance, but I’m unsure if numbers get reused from one locality to another.

Reply:
We might consider a more flexible map key scheme, such as: https://www.npmjs.com/package/regexp-map

Reply:
If the local route numbering system is also uniform across the country, maybe we could see if mappers in China would be open to more generic `network` values.